ui/app/js/controllers/user_edit_controller.js

```diff
@@ -148,7 +152,7 @@ function ($scope, $modalInstance, CSRF, Permissions, user, users) {
       })
       .error(function(response) {
         if (typeof response === 'object') {
-          $scope.errors = response;
+          $scope.errors.permission = response;
```

Comment:
It sounds like there might be another bug if there's a case where "No JSON object decoded" is shown...